go.mod

```diff
@@ -1,19 +1,21 @@
 module github.com/databricks/terraform-provider-databricks
 
-go 1.22
+go 1.22.0
+
+toolchain go1.22.5
```

Comment:
curious -- do you know why didn't we had this before?

Reply:
Not sure why we didn't have it, it could be change in the recent `go mod` behaviour - now it always added and before it didn't happen